pkgs/by-name/fc/fcitx5-nord/package.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "fcitx5-nord";
+  version = "unstable-20210727";
```

Comment:
`version` here should be `unstable-2021-07-27`

<https://nixos.org/manual/nixpkgs/stable/#sec-package-naming>
_If a package is not a release but a commit from a repository, then the version attribute must be the date of that (fetched) commit. The date must be in "unstable-YYYY-MM-DD" format._